### ISSUE TITLE
Problem Object API\models\research\object_detection>python train.py --train_dir=training/ --pipeline_config_path=training/ssd_mobilenet_v1_pets.config --logtostderr

### ISSUE THREAD

**Issue by Homayra22** [tensorflow/models#9705]
(object) C:\Users\homay\OneDrive\Documents\Object API\models\research\object_detection>python train.py --train_dir=training/ --pipeline_config_path=training/ssd_mobilenet_v1_pets.config --logtostderr
Traceback (most recent call last):
  File "C:\Users\homay\OneDrive\Documents\Object API\models\research\object_detection\train.py", line 48, in <module>
    import tensorflow as tf
ModuleNotFoundError: No module named 'tensorflow'

**Comment by ravikyram** [comment]
@Homayra22 

 What is your cpu make/model?
I suspect that your cpu does not support AVX instructtion sets.See [hardware requirements](https://www.tensorflow.org/install/pip#hardware-requirements).
Check if your Python is for 32- or 64-bit installation?. IT should be 64 bit.
Also, can you create a fresh environment and then try to install Tensorflow. and let us know if the issue still persists. Thanks!

**Comment by Homayra22** [comment]
My CPU is Corei5, 8 gm RAM. 
I am using Tensor flow 1.12
My Python is  3.8- 64 bit
I am trying to create a new environment for my anaconda tensor flow. In the anaconda navigator the  available tensor flow package is 2.3.0 but I require 1.12 or 1.15. Can you please help me how can I install tensor flow 1.12 or 1.15 in anaconda navigator?

**Comment by ravikyram** [comment]
@Homayra22 

Please, see tested build configurations for [windows](https://www.tensorflow.org/install/source_windows#cpu) and [linux](https://www.tensorflow.org/install/source#cpu) . Please try with python 3.6 and see if the issue still persists. Thanks!

**Comment by Homayra22** [comment]
@ravikyram 

Hi, After trying here for so long hours, I have moved to another tutorial. I am facing a problem when training a custom model

tensorflow.python.framework.errors_impl.InvalidArgumentError: NewRandomAccessFile failed to Create/Open: C:\Tensorflow\workspace        raining_demonnotations\label_map.pbtxt : The filename, directory name, or volume label syntax is incorrect.
; no protocol option


I have already checked label_map.pbtxt file. The command is written properly.
